common/script/ops/scoreTask.js

```diff
@@ -87,7 +87,6 @@ function _gainMP (user, val) {
   if (user.stats.mp >= user._statsComputed.maxMP) user.stats.mp = user._statsComputed.maxMP;
   if (user.stats.mp < 0) {
     user.stats.mp = 0;
-    return user.stats.mp;
```

Comment:
I think you're right. I don't see anywhere in the code where the result of the function is assigned to a value.